packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -92,8 +92,6 @@ export default function ScheduleDetails() {
     return state.prefs.local.dateFormat || 'MM/dd/yyyy';
   });
 
-  let pageType = usePageType();
```

Comment:
🧹  Unused

Reply:
Nice! We should really consider adding a eslint rule to catch these automatically.